test/threads_exec.jl

```diff
@@ -53,6 +53,26 @@ if threadpoolsize() > 1
     end
 end
 
+if threadpoolsize() > 1
+    let lk = PaddedSpinLock()
```

Comment:
```suggestion
    let lk = Base.Threads.PaddedSpinLock()
```

I think this was the cause of the only test failure